components/brave_wallet/common/common_utils.cc

```diff
@@ -28,13 +28,15 @@ namespace brave_wallet {
 namespace {
 
 bool IsDisabledByPolicy(PrefService* prefs) {
-#if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_LINUX)
+#if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_LINUX) || \
+    BUILDFLAG(IS_IOS)
   DCHECK(prefs);
   return prefs->IsManagedPreference(prefs::kDisabledByPolicy) &&
          prefs->GetBoolean(prefs::kDisabledByPolicy);
 #else
   return false;
-#endif  // BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_LINUX)
+#endif  // BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_LINUX) ||
+        // BUILDFLAG(IS_IOS)
 }
 
```

Comment:
It seems we don't need these platform cheecks at all. On android `prefs->GetBoolean(prefs::kDisabledByPolicy)` this would be always false

Reply:
Yeah looks like the pref is registered regardless of platform, so removed these buildflag checks

Reply:
@supermassive I had to add the buildflag check for android back here since the unit test above relies on the fact that even after setting the managed pref the IsDisabledByPolicy method should still return false: https://github.com/brave/brave-core/pull/30089/commits/96f766757a978e95d34136a7da0499958db6c2fe